File: ios/RnFluidicSlider.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>
#import <React/UIView+React.h>

@interface RCT_EXTERN_MODULE(RnFluidicSlider, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(onSlideStart, RCTDirectEventBlock)
RCT_EXPORT_VIEW_PROPERTY(onSlideEnd, RCTDirectEventBlock)

@end

File: ios/RnFluidicSlider.swift
@objc(RnFluidicSlider)
class RnFluidicSlider: RCTViewManager {    
    lazy var sliderView = RnFluidicSliderView()
    
    override func view() -> UIView! {
        return sliderView
    }
    
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
}
